src/coord_seq.rs

```diff
@@ -149,6 +149,93 @@ impl<'a> CoordSeq<'a> {
         }
     }
 
+    #[cfg(any(feature = "v3_10_0", feature = "v3_11_0"))]
+    pub fn new_from_buffer<T: AsRef<[f64]>>(
+        data: &[f64],
+        size: u32,
+        dims: u32,
+        has_z: bool,
+        has_m: bool,
+    ) -> GResult<CoordSeq<'a>> {
+        let data_ptr = &data[0] as *const f64;
```

Comment:
```suggestion
        let data_ptr = data.as_ptr();
```

https://doc.rust-lang.org/std/primitive.slice.html#method.as_ptr

Reply:
I knew there had to be a simpler way 😂